### ISSUE TITLE
Tab indents code correctly, but SHIFT + tab does not remove indentation

### ISSUE THREAD

**Issue by guymarshall** [lapce/lapce#1068]
## Lapce Version

0.2.0

## System information

Windows 10 x64

## Describe the bug

Indentation works for indenting but not for removing indentation.

## Additional information

- screenshots
- logs
- theories about what might be going wrong
- workarounds that you used

**Comment by snirt** [comment]
This issue happens in MacOS and Linux as well

**Comment by hakro** [comment]
I had the same situation as well.
Indent with `Tab` works, but not `Shift` + `Tab` to go back.

But I checked in the Keybinding setting, and I found both actions mapped to `Ctrl` + `[` or ']`
So I remapped them back to what the conventional shortcuts, and that worked.

![2023-05-21_16-30](https://github.com/lapce/lapce/assets/3624853/9d0ad7da-fd35-47c1-8e90-3f5bedcc96a5)

I do think that `Shift` + `Tab` should be the default though. That's the way it works everywhere else.
